src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -157,7 +161,8 @@ public function remove($files)
                     }
                 } else {
                     if (true !== @unlink($file)) {
-                        throw new IOException(sprintf('Failed to remove file %s', $file));
+                        $error = error_get_last();
+                        throw new IOException(sprintf('Failed to remove file "%s": %s.', $file, $error['message']));
```

Comment:
Usually, this kind of changes are considered as new features, to me moved to a PR on the master branch IMO